app/commands/exercise/representation/search.rb

```diff
@@ -2,7 +2,7 @@ class Exercise::Representation::Search
   include Mandate
 
   # Use class method rather than constant for easier stubbing during testing
-  def self.requests_per_page = 20
+  def self.requests_per_page = 21
```

Comment:
```suggestion
  def self.requests_per_page = 20
```

I don't know how this keeps getting included everywhere! :)